### PR TITLE
Language update for customized post processor

### PR DESCRIPTION
**Pull request by peterennis**

### CODE REVIEW

File: Documentation/tutorial/howto_add_a_customized_post_processor.md [hunk 1]
```diff
@@ -49,20 +49,20 @@ Take `ExtractSearchIndex` for example, we add `"_enableSearch": true` in global
...
-Take `ExtractSearchIndex` for example again, we traverse all HTML files, extract key words from these HTML files and save a file named `index.json` under the `outputFolder`. At last, we return the manifest which is not modified.
+Using `ExtractSearchIndex` for example again, we traverse all HTML files, extract key words from these HTML files and save a file named `index.json` under the `outputFolder`. Finally we return the manifest which is not modified.
 
 ## Step4: Build your project and copy the output dll files to:
 
 * Global: the folder with name `Plugins` under DocFX.exe
 * Non-global: the folder with name `Plugins` under a template folder, then run `DocFX build` command with parameter `-t {template}`.
 
-    *Hint*: DocFX can merge templates, so we can specify multiple template folders as `DocFX build -t {templateForRender},{templateForPlugins}`. Each of template folder should have a subfolder named `Plugins` with exported assemblies.
+    *Hint*: DocFX can merge templates, so we can specify multiple template folders as `DocFX build -t {templateForRender},{templateForPlugins}`. Each of teh template folders should have a subfolder named `Plugins` with exported assemblies.
```
Comment (vwxyzh) [inline Documentation/tutorial/howto_add_a_customized_post_processor.md]: typo.

**Comment by vwxyzh** [comment]
:+1:

**Comment by superyyrrzz** [comment]
:+1:

**Comment by qinezh** [comment]
:+1:

**Pull request merged**